src/main/java/com/bookstore/controller/BookController.java

```diff
@@ -15,34 +18,40 @@
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
+@Tag(name = "Book management", description = "Endpoints for managing books")
 @RequiredArgsConstructor
 @RestController
 @RequestMapping(value = "books")
 public class BookController {
     private final BookService bookService;
 
     @GetMapping
-    public List<BookDto> getAll() {
-        return bookService.findAll();
+    @Operation(summary = "Get all books", description = "Get a list of all available books")
+    public List<BookDto> getAll(Pageable pageable) {
+        return bookService.findAll(pageable);
     }
 
     @GetMapping("/{id}")
+    @Operation(summary = "Get book by id", description = "Get book by id")
```

Comment:
I think we don't need this duplication of text in summary and description. Its better to remove one of them. Please do it for every next method

Reply:
Good point, you can put to the summary just sth like this: 
- Get a book
- Get books
- Create a book
- Update a book
- etc.